### PR TITLE
Swagger docs

### PR DESCRIPTION
**Pull request by jsstevenson**
Add/change description text as it appears on the root docs page

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -19,36 +19,49 @@ def custom_openapi():
...
-    #     "email": "his email"
-    # }
+    openapi_schema['info']['license'] = {  # TODO
+        "name": "name of license",
+        "url": "link"
+    }
+    openapi_schema['info']['contact'] = {  # TODO
+        "name": "Alex Wagner",
+        "email": "email@email.com"
+    }
```
Comment (korikuzma) [inline main.py:32]:
```suggestion
# TODO: Figure out how to add these back
    # openapi_schema['info']['license'] = {  # TODO
    #     "name": "name of our license",
    #     "url": "link to it"
    # }
    # openapi_schema['info']['contact'] = {  # TODO
    #     "name": "alex wagner",
    #     "email": "his email"
    # }
```

File: main.py [hunk 2]
```diff
@@ -19,36 +19,49 @@ def custom_openapi():
...
     app.openapi_schema = openapi_schema
     return app.openapi_schema
 
 
 app.openapi = custom_openapi
 
+# endpoint parameter description text
+read_query_summary = "Given query, provided matches and match ratings from "\
+                     "aggregated sources."
```
Comment (korikuzma) [inline main.py:41]:
```suggestion
read_query_summary = "Given query, provide highest matches from "\
                     "aggregated sources."
```

File: main.py [hunk 3]
```diff
@@ -19,36 +19,49 @@ def custom_openapi():
...
+response_description = "A response to a validly-formed query."
+q_descr = "String to normalize."
```
Comment (korikuzma) [inline main.py:43]:
```suggestion
q_descr = "Therapy to normalize."
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
Comment out adding info for now.

**Review by korikuzma — APPROVED** [review]

File: main.py [hunk 4]
```diff
@@ -19,36 +18,62 @@ def custom_openapi():
     openapi_schema = get_openapi(
         title="Therapy Normalizer",
         version="0.1.0",
-        description="TODO Description here",  # TODO
+        openapi_version="3.0.3",
+        description="Normalize therapy terms.",
```
Comment (ahwagner) [inline main.py:22]:
```suggestion
        description="Normalize drugs and other therapy terms.",
```

File: main.py [hunk 5]
```diff
@@ -19,36 +18,62 @@ def custom_openapi():
     openapi_schema = get_openapi(
         title="Therapy Normalizer",
```
Comment (ahwagner) [inline main.py:19]:
```suggestion
        title="The VICC Therapy Normalizer",
```

File: main.py [hunk 6]
```diff
@@ -19,36 +18,62 @@ def custom_openapi():
...
-    #     "url": "link to it"
-    # }
-    # openapi_schema['info']['contact'] = {  # TODO
-    #     "name": "alex wagner",
-    #     "email": "his email"
-    # }
+    openapi_schema['info']['license'] = {  # TODO
+        "name": "Name-of-license",
+        "url": "http://www.to-be-determined.com"
+    }
```
Comment (ahwagner) [inline main.py:28]:
```suggestion
#    openapi_schema['info']['license'] = {  # TODO
#        "name": "Name-of-license",
#        "url": "http://www.to-be-determined.com"
#    }
```

Let's not attach a license for now.

File: main.py [hunk 7]
```diff
@@ -19,36 +18,62 @@ def custom_openapi():
...
+    openapi_schema['info']['contact'] = {  # TODO
+        "name": "Alex Wagner",
+        "email": "email@email.com"
+    }
```
Comment (ahwagner) [inline main.py:32]:
```suggestion
    openapi_schema['info']['contact'] = {  # TODO
        "name": "Alex H. Wagner",
        "email": "Alex.Wagner@nationwidechildrens.org"
    }
```

File: main.py [hunk 8]
```diff
@@ -19,36 +18,62 @@ def custom_openapi():
...
-               keyed: Optional[bool] = False,
-               incl: Optional[str] = '',
-               excl: Optional[str] = ''):
+         response_description=response_description,
+         response_model=Service)
+def read_query(q: str = Query(..., description=q_descr),
+               keyed: Optional[bool] = Query(False, description=keyed_descr),
+               incl: Optional[str] = Query('', description=incl_descr),
+               excl: Optional[str] = Query('', description=excl_descr)):
+    """Normalize query string given by user.
```
Comment (ahwagner) [inline main.py:65]:
```suggestion
    """Return strongest match concepts to query string provided by user.
```

**Review by ahwagner — CHANGES_REQUESTED** [review]
Minor edits. Looks great.

**Pull request merged**